freight_bringup/launch/include/aggregator.launch.xml

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <node 
+        pkg="diagnostic_aggregator" type="aggregator_node"
```

Comment:
need to add a runtime dependency on diagnostic_aggregator for both fetch/freight_bringup packages, else this will break the world